### ISSUE TITLE
Gerber files won't open

### ISSUE THREAD

**Issue by bdagenais** [beyerch/DHP_Kepler#1]
I tried opening the gerber files with Altium and Viewmate with no luck.

Could it be possible to look into this?

Thanks !

**Comment by beyerch** [comment]
Which file were you trying to open?  I just opened the KeplerAdvancedVehicleinterface.SchDoc without any issue with Altium Viewer.

**Comment by TheMutley** [comment]
Issue resolved! Any ETA for the firmware side ?

**Comment by bdagenais** [comment]
Got it to work, issue was on my side.

Thanks!

**Comment by beyerch** [comment]
Cool, glad you got it to work.  I opened some of them in a viewer last night to double check and didn’t have issues either.

thx

**Comment by TheMutley** [comment]
@beyerch do you have an ETA for the firmware release ?

**Comment by beyerch** [comment]
I'll get that and J2535 posted by the weekend.. been traveling for work.

Sent from my iPhone

On Jun 14, 2018, at 2:39 PM, Mutley <notifications@github.com<mailto:notifications@github.com>> wrote:


@beyerch<https://github.com/beyerch> do you have an ETA for the firmware release ?

**Comment by beyerch** [comment]
Firmware is pushed.

J2534 is pushed.

3d enclosure is pushed.

I need to do some cleanup, especially with the J2534 stuff, though.

Thanks

**Comment by bdagenais** [comment]
Thanks for the update.

iam having a hard time finding the OBD connector...

Would you have a source/ part number?

It is not in the BOM.

I have updated the part number in your BOM to suit parts availability, I will sen dit back when done.

**Comment by beyerch** [comment]
Yeah, I have a P/N somewhere.  That was a bit of a pain to find….  Let me dig around for it.

**Comment by beyerch** [comment]
On a side note, you involved with ECU tuning on Subaru, etc?   I’m also working on tuning software, but primarily focus on GM, though looking to expand.  Curious if there are any opportunities to work together there commercially / open source.

**Comment by bdagenais** [comment]
Hi there again!
Sorry I did not reply sooner.

I have worked in the Subaru tuning scene for 15 years. Send me your e-mail and we can chat off GitHub about it.

I have found the OBDII connector on Aliexpress, I will send you the updated BOM later today.

Thanks again!

**Comment by beyerch** [comment]
Charles@charlescbeyer.com<mailto:Charles@charlescbeyer.com>

Thx!

Sent from my iPhone

On Jul 9, 2018, at 9:52 AM, bdagenais <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi there again!
Sorry I did not reply sooner.

I have worked in the Subaru tuning scene for 15 years. Send me your e-mail and we can chat off GitHub about it.

I have found the OBDII connector on Aliexpress, I will send you the updated BOM later today.

Thanks again!
